### ISSUE TITLE
[Nest] { Error: UNAUTHORIZED:

### ISSUE THREAD

**Issue by travisjaymoore** [KraigM/homebridge-nest#146]
I'm in way over my head. Followed all the instructions to the letter, the JSON file checks out, but I'm getting this string of errors and have no idea what to do next.
I double checked my ClientID, Client Secret and pulled another code from Nest. I even added the legacy username and password and still now luck.

![screen shot 2017-10-30 at 4 56 13 pm](https://user-images.githubusercontent.com/33238188/32200278-765341e4-bd96-11e7-8a0c-e92dcaa0ef87.png)

**Comment by gmaster12** [comment]
I have the exact same error too. I did the same as you, generated another code just in case. config.json passes the checker as well. Maybe something has changed in the API. I noticed the docs here call out read/write v4 and in the developer sign on they have read/write v6.

**Comment by gmaster12** [comment]
Just figured it out. Make sure the token is EMPTY like so:

"token": "",

Before it had an example token in there. If you don't take it out it assumes that's the right one and you'll get an authorization failure.  Regenerate a new Key then try again. Mine worked fine after emptying the token line.

**Comment by remcovdwal** [comment]
That saved me! Thought it had something to do with write permissions, but it gladly didnt.

Thnx man!

**Comment by larrybeck64** [comment]
Yep, same issue caught me. Might warrant an update in the install guide?

**Comment by maartenvant** [comment]
Same issue here, above solved it for me. might be a good idea to add this to the installation doc
